cheng-k/2025-Summer.md

```diff
@@ -6,32 +6,83 @@
 
 #### 🌹
 
-* 
+* I now have a good understanding about BIDS, goals it strives to address, and ideas it stands for. I am looking forward to continuing to work on these initiatives!
 
 #### 🌱
 
-* 
+* I have learned a lot about open source in the past few days. I am amazed at the quality of work put in by passionate volunteers. I never realized that there needs to be documentation on how to write documentation. So many products use open source tools such as Python, and it's all thanks to the collective work of thousands (millions?) of people who together want to create a useful and accessible tool. Managing a repository and its growing community is no small feat behind the scenes.
 
-#### 🌵   
+#### 🌵
 
-* 
+* The "The Turing Way" book is written with great depth. It is sometimes difficult to recognize what information can be added simply due to the fact that it is already so detailed. Some possible additions are already written elsewhere in the book, so I am not sure if it is a matter of linking to it or adding a copy.
 
 ### 🤝 Reciprocal support
 
 #### Who did I help?
 
-* 
+* Trying to identify ways to (further) improve The Turing Way book
 
 #### Who helped me?
 
-* 
+* KW
+  * Introduced me to BIDS and laid foundation for next few weeks
+  * Gave tour of AI Futures space
+  * Provided background and motivation for BIDS, open source, and open science.
+  * Suggested areas in The Turing Way book that could be improved and explained how to navigate the repository/book
+  * Assigned resources (videos, existing written documentation) to consult
+  * Create github issue on The Turing Way repository
+* JK
+  * Granted cory building keycard access and tour
+  * Contacted IT department as I was having difficulty logging into my Google account (resolved)
+  * Intro to being an intern here
+* LWH
+  * BIDS suite: keycard access, tour, and desk assignment
+  * Sent me suite welcome email, requested my title+headshot+bio (todo)
 
 ### Activities / tasks
 
 #### What did I do?
 
-* 
+##### Main
+
+* Watched [Stefan's talk](https://www.youtube.com/watch?v=cmUtiEv4xm4) from last year and took notes (3 hours)
+  * Good intro to the community behind open science, can elaborate more from this video if needed.
+  * I am really surprised that 50% of UC Berkeley undergraduates take Data8 - Intro to Data Science!
```

Comment:
Move this section to its own section called `##### Learning about open source` or something similar?

I'm so glad that you watched this - I think there's loads of content in there that I'm keen for you and Tyler to create some blog posts / social media posts from over the summer ☀️